### PR TITLE
Revert "Add forcedfocus on enterEditing"

### PR DESCRIPTION
**Pull request by asturur**
Reverts kangax/fabric.js#1961

### CODE REVIEW

**Comment by kangax** [comment]
Do we need this on?

**Comment by asturur** [comment]
absolutely.
This was here for a reson, and it is not related to android keyboard as in the comment.
EnteringEditing we put focus on the hiddenTextarea, and this was an improvement.
Removing this function that we are reverting, causes the focus  to disappear as soon as we click again the iText object in the editing mode.
Because it won t trigger another enterEdit, and the focus will go away from hidden Text Area.

File: src/mixins/itext_behavior.mixin.js [hunk 1]
```diff
@@ -335,7 +335,6 @@
       this.isEditing = true;
 
       this.initHiddenTextarea();
-      this.hiddenTextarea.focus();
```
Comment (asturur) [inline src/mixins/itext_behavior.mixin.js]: wait this has to stay.

**Comment by kangax** [comment]
@asturur can you plz merge yourself once it's ready

**Pull request merged**